packages/create-app/src/lib/bin.ts

```diff
@@ -296,7 +308,10 @@ function createConfig(
   platforms: TemplateInfo[],
   plugins: TemplateInfo[] | null,
   bundler: TemplateInfo,
-  remoteCacheProvider: SupportedRemoteCacheProviders | null,
+  remoteCacheProvider: {
+    name: SupportedRemoteCacheProviders;
+    args: Record<string, unknown>;
```

Comment:
should I add the types for args into the `tools` package, adding deps on both provider packages and inferring their types?

Reply:
nah let's keep these generic, as people may add custom providers we won't have access to